server/server_test.go

```diff
@@ -44,3 +44,120 @@ func TestCreateUserHandler(t *testing.T) {
 		t.Fatalf("Result is not correct!!!")
 	}
 }
+
+func TestGetUserInfoHandler(t *testing.T) {
+
+	srv := &Server{}
+	srv.Users.U = make(map[int]pkg.User)
```

Comment:
let's not check against an empty user, but populate this map at first and then check against populated user